src/Context/LoginContext.php

```diff
@@ -80,6 +81,45 @@ public function stepIAmNotLoggedIn()
      * @param string $password
      */
     public function stepILogInWith($email, $password)
+    {
+        $this->loginWith($email, $password);
+
+        // Check if MFA module is installed
+        if (!class_exists(RegisteredMethod::class)) {
+            return;
+        }
+
+        // Skip MFA registration if MFA module installed
+        $this->getMainContext()->getSession()->wait(300);
+        $page = $this->getMainContext()->getSession()->getPage();
+        $mfa = $page->find('css', '#mfa-app');
+        if (!$mfa) {
+            return;
+        }
+        $btns = $page->findAll('css', '.mfa-action-list__item .btn');
+        foreach ($btns as $btn) {
+            if ($btn->getText() !== 'Setup later') {
+                continue;
```

Comment:
Sound like we should fail the behat build if we can't find the "Setup later" button.

Reply:
Have added an assertion